lib/constants/platforms.ts

```diff
@@ -1,19 +1,21 @@
-export const PLATFORM_TYPE_AZURE = 'azure';
-export const PLATFORM_TYPE_BITBUCKET = 'bitbucket';
-export const PLATFORM_TYPE_BITBUCKET_SERVER = 'bitbucket-server';
-export const PLATFORM_TYPE_GITEA = 'gitea';
-export const PLATFORM_TYPE_GITHUB = 'github';
-export const PLATFORM_TYPE_GITLAB = 'gitlab';
+export enum PlatformID {
```

Comment:
@JamieMagee i think const enum is better here, so it will be eliminated at runtime?

I think it should be `PlatformId` or `PlatformType` instead.

Reply:
Yeah, good idea with `const enum` 👍